src/MicrosoftTeams.ts

```diff
@@ -134,6 +304,10 @@ namespace microsoftTeams {
         Plc = 3,
         Staff = 4,
     }
+    interface Window {
```

Comment:
nit: need a newline before this interface.

Reply:
done